confdb/aclocal_cc.m4

```diff
@@ -1651,3 +1651,31 @@ AC_DEFUN([PAC_C_STATIC_ASSERT], [
     AC_MSG_RESULT([no])
     ])
 ])
+
+dnl
+dnl PAC_CC_CHECK_TLS - Test for thread local storage support
+dnl
+dnl will AC_DEFINE([TLS]) to a compiler supported TLS keyword
+dnl
+AC_DEFUN([PAC_CC_CHECK_TLS], [
+    AC_MSG_CHECKING([for thread-local storage])
+    if test -z $pac_cv_tls ; then
```

Comment:
Maybe we should use `AC_CACHE_CHECK` here?

Reply:
Done.